packages/chip/src/lib/Chip.tsx

```diff
@@ -8,9 +8,9 @@ export type ChipProps = {
   size?: 'small' | 'medium';
```

Comment:
now that we've split it into 2 components we can get rid of size and color.

Reply:
removed